scripts/generate_stubs.py

```diff
@@ -102,7 +102,7 @@ def includes_op(self, op: str) -> bool:
         },
     ),
     GroupedOpCodes(
-        name="Box",
+        name="BoxStatic",
```

Comment:
This might not be necessary now that we're splitting the ops into a separate module